src/pages/TradeHistory/index.tsx

```diff
@@ -46,6 +52,18 @@ const TradeHistory: React.FC<TradeHistoryProps> = ({ swaps }) => {
     );
   };
 
+  const copyTxId = (txid: string) => {
```

Comment:
I would concat the `blockstream.info/liquid/tx` base URL so its more handy to copy paste in a browser